homeworks/oleg.dzhygil_dzhygil/homework_7/fibonacci.task.js

```diff
@@ -0,0 +1,14 @@
+function fibonacci() {
+    const random = Math.floor(Math.random() * 50 + 50);
+    const array = [0, 1];
+    for (let i = 0; i < random; i++) {
```

Comment:
your loop will do lots of unneeded iterations, this is more suitable case for a `while` loop